src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {
```

Comment:
```suggestion
public interface Drawable {
```
not only figures can be drawn